SAMtags.tex

```diff
@@ -453,6 +455,112 @@ \subsubsection{Color space}
+below, taken from \emph{Modeling methyl-sensitive transcription factor
+motifs with an expanded epigenetic alphabet}, Coby Viner
+et.al. \url{https://www.biorxiv.org/content/10.1101/043794v1}.
+Additionally ambiguity codes {\tt A}, {\tt C}, {\tt G} and {\tt T}
+exist to represent unspecified modifications bases of their respective
+canonical base types, plus code {\tt N} to represent an unspecified
+modification of any base type.
+
+\begin{center}
+\begin{tabular}{lllll}
+{\bf Unmodified base} & {\bf Code} & {\bf Abbreviation} & {\bf Name} & {\bf ChEBI} \\
+\hline
+C & m & 5mC   & 5-Methylcytosine        & 27551 \\
+C & h & 5hmC  & 5-Hydroxymethylcytosine & 76792 \\
+C & f & 5fC   & 5-Formylcytosine        & 76794 \\
+C & c & 5caC  & 5-Carboxylcytosine      & 76793 \\
+C & C &       & Ambiguity code; any C mod & \\
+\hline
+T & g & 5hmU  & 5-Hydroxymethyluracil   & 16964 \\
+T & e & 5fU   & 5-Formyluracil          & 80961 \\
+T & b & 5caU  & 5-Carboxyluracil        & 17477 \\
+T & T &       & Ambiguity code; any T mod & \\
+\hline
+A & a & 6mA   & 6-Methyladenine         & 28871 \\
+A & A &       & Ambiguity code; any A mod & \\
+\hline
+G & o & 8oxoG & 8-Oxoguanine            & 44605 \\
+G & G &       & Ambiguity code; any G mod & \\
+\hline
+N & n & Xao   & Xanthosine              & 18107 \\
+N & N &       & Ambiguity code; any mod & \\
+\end{tabular}
+\end{center}
+
+\item[MP:Z:\tagvalue{qualities}]
+\hfill\\
+The optional {\tt MP} tag lists the Phred qualities of each modification listed in the {\tt MM} tag.
+The qualities are encoded in the same manner as the primary {\sf QUAL} field; one byte per quality with ASCII value Phred score + 33.
+A space character (`{\tt \textvisiblespace}') should be used as a separator between concatenated quality strings when multiple modification lists are present in the {\tt MM} tag.
+The length should match the number of position deltas from {\tt MM} plus 1 per space character required.
+
+For example {\tt MM:Z:C+m,5,12,3;C+h,57;} may have an associated
+quality tag of {\tt MP:Z:5EB /}.
+
+Quality values for ambiguity codes give the likelihood that the
+modification is one of the possible codes compatible with that
+ambiguity code.  For example {\tt MM:Z:C+C,10; MP:Z:+} indicates a C
+call with an unspecified modification and the phred score of 10 (ASCII
+value {\tt +}).  This corresponds to a 90\% chance of the base being
+modified.
+
+To represent several possible modifications at the same site the {\tt MP} tag can be used to indicate the probabilities of each possibility.
+The values used should be absolute probabilities, not relative between the alternatives.
+For example, a C base that has 90\% chance of being modified with 5mC being three times more likely than 5hmC will encode 5mC with 67.5\% probability ($0.9 * 0.75$)and 5hmC with 22.5\% probability ($0.9 * 0.25$).
+This could be represented with {\tt MM:Z:C+m,10;C+h,10; MP:Z:" \&}.
```

Comment:
it would be good to quote this or other way separate from the period at the end of the sentence, as it runs together in the PDF (also on line 499 above, though there the semicolon acts as a implicit separator.